integration_tests/optout.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from '@playwright/test'
+import tokenVerification from './mockApis/tokenVerification'
+import { resetStubs } from './mockApis/wiremock'
+import feComponent from './mockApis/feComponent'
+import assessForEarlyRelease from './mockApis/assessForEarlyRelease'
+import { favicon, getSignInUrl, signIn, signOut, token } from './mockApis/auth'
+
+const baseUrl = 'http://localhost:3007'
```

Comment:
suggestion (non-blocking): I think there's an [option to set a base url](https://playwright.dev/docs/test-configuration#:~:text=use%3A%20%7B%0A%20%20%20%20//%20Base%20URL%20to%20use%20in%20actions%20like%20%60await%20page.goto(%27/%27)%60.%0A%20%20%20%20baseURL%3A%20%27http%3A//127.0.0.1%3A3000%27%2C)

Reply:
good that will be much better